### ISSUE TITLE
Unable to enable "Approve all"-button in CMP notice as VoiceOver user

### ISSUE THREAD

**Issue by johankasperi** [didomi/react-native#112]
When you are scrolling to the bottom of the notice as a VoiceOver user (using three fingers) the "Approve all"-button is not enabled. Making it impossible to close the notice and access our app. 

https://github.com/didomi/react-native/assets/7563261/6ec8ee2d-34d9-4050-9700-03612c1d67bd

Also, the accessibility state of the button is not set to "disabled" making it impossible for a VoiceOver user to understand that it is disabled. Using only visual feedback for when a button is disabled is not enough from an accessibility perspective.

**Comment by pmerlet-at-didomi** [comment]
Hello @johankasperi , this issue should be fixed since version 2.7.0, can you check on your side?

**Comment by johankasperi** [comment]
Yes seems to be working now. Thank you!
